Handles a `PushRepository.DoesNotExist` exception (#1727)
pulp/pulp_container: docs/user/guides/push-image.md

!!! note

    Content is pushed to a push repository type. A push repository does not support mirroring of the

>     remote content via the Pulp API. Trying to push a content with the same name of an existing
>     "regular" repository will fail.

Reviewer: ```suggestion
    remote content via the Pulp API. Trying to push content with the same name as an existing
    "regular" repository will fail.
```
